lib/internal/errors.js

```diff
@@ -755,7 +755,7 @@ E('ERR_CRYPTO_SCRYPT_NOT_SUPPORTED', 'Scrypt algorithm not supported', Error);
 // Switch to TypeError. The current implementation does not seem right.
 E('ERR_CRYPTO_SIGN_KEY_REQUIRED', 'No key provided to sign', Error);
 E('ERR_CRYPTO_TIMING_SAFE_EQUAL_LENGTH',
-  'Input buffers must have the same length', RangeError);
+  'Input buffers must have the same number of bytes', RangeError);
```

Comment:
Totally fine the way it is, but maybe consider this?:

```suggestion
  'Input buffers must have the same byte length', RangeError);
```

Reply:
I am happy to change it. This was the last suggestion in https://github.com/nodejs/node/pull/23341#discussion_r235178956.

Reply:
I prefer "byte length", too